src/AppInstallerCommonCore/Public/winget/ManifestValidation.h

```diff
@@ -249,6 +251,8 @@ namespace AppInstaller::Manifest
         bool m_warningOnly;
     };
 
+    static const std::unordered_set<std::filesystem::path> allowedPortableFiletypes = { L".exe" };

```

Comment:
nit: AllowedPortableFiletypes   or   s_AllowedPortableFiletypes  for static const

Reply:
Fixed